### PR TITLE
posibrains now get admin set policy instead of code set welcome message

### PR DESCRIPTION
**Pull request by Fikou**
danse caremanel dasenn ijtsjojaog metkoa karamel danse

## Changelog
:cl:
admin: posibrains now use get_policy for their laws
/:cl:

### CODE REVIEW

**Comment by TheSilverNuke** [comment]
I do agree that Posibrains need to be updated. However in this situation they'll just smack someone over a small escalation if in a mech and say "I did my best but he kept coming after me," in regards to the broken spessman bleeding on the ground.

Instead I would tweak it to say, "Follow the directives given to you by your activator," this way if they are activated by normal crew they can just do crew stuff. If they are activated by a tot they will help him too. It clears any confusion about posibrain policy and helps direct the flow of the round.

**Comment by HomocidlePancakes** [comment]
Just give them laws already.

**Comment by zxaber** [comment]
I don't really think there's a need to bind posis to their creator like golems or pAIs are. If you need the posi to follow your orders (tot or otherwise), just make them into an AI and card them into the mech.

**Comment by Tlaltecuhtli** [comment]
what about a press in hand to give a directive and if left empty they are the same as purged ai (obey server rules + NAP mode)

**Comment by Tharcoonvagh** [comment]
>ship

>space station 13

**Review by ExcessiveUseOfCobblestone — CHANGES_REQUESTED** [review]
https://github.com/tgstation/tgstation/pull/50185#issuecomment-603876333

File: code/modules/mob/living/brain/posibrain.dm [hunk 1]
```diff
@@ -23,11 +23,6 @@ GLOBAL_VAR(posibrain_notify_cooldown)
 	var/fail_message = "<span class='notice'>The positronic brain buzzes quietly, and the golden lights fade away. Perhaps you could try again?</span>"
 	///Role assigned to the newly created mind
 	var/new_role = "Positronic Brain"
-	///Message sent to the player possessing the brain
-	var/welcome_message = "<span class='warning'>ALL PAST LIVES ARE FORGOTTEN.</span>\n\
```
Comment (spookydonut) [inline code/modules/mob/living/brain/posibrain.dm]: dont you need to keep a default message if no policy is set

Comment (Fikou) [reply code/modules/mob/living/brain/posibrain.dm]: idk cobby told me to delete it

Comment (ExcessiveUseOfCobblestone) [reply code/modules/mob/living/brain/posibrain.dm]: i told headmins they need to update it

**Comment by ExcessiveUseOfCobblestone** [comment]
i was told headmins did this

**Pull request merged**